webroot/adm/client-key.html

```diff
@@ -82,17 +82,6 @@ <h3>Output</h3>
             doApiCall('POST', url, '#standardOutput', '#errorOutput');
         });
 
-        $('#doDel').on('click', function () {
```

Comment:
Why?

Reply:
We don't support client delete anymore, should we still keep it?